docs/standard/garbage-collection/using-objects.md

```diff
@@ -14,47 +14,50 @@ ms.assetid: 81b2cdb5-c91a-4a31-9c83-eadc52da5cf0
-  
+Instead of wrapping a `try/finally` block in a `using` statement, you may choose to implement the `try/finally` block directly. This may be your personal coding style, or you might want to do this for one of the following reasons:
+
+- To include a `catch` block to handle any exceptions thrown in the `try` block. Otherwise, any exceptions thrown by the `using` statement are unhandled, as are any exceptions thrown within the `using` block if a `try/catch` block isn't present.
+
+- To instantiate an object that implements <xref:System.IDisposable> whose scope is not local to the block within which it is declared.
+
+The following example is similar to the previous example, except that it uses a `try/catch/finally` block to instantiate, use, and dispose of a <xref:System.IO.StreamReader> object, and to handle any exceptions thrown by the <xref:System.IO.StreamReader> constructor and its <xref:System.IO.StreamReader.ReadToEnd%2A> method. Note that the code in the `finally` block checks that the object that implements <xref:System.IDisposable> isn't `null` before it calls the <xref:System.IDisposable.Dispose%2A> method. Failure to do this can result in a <xref:System.NullReferenceException> exception at run time.
+
 [!code-csharp[Conceptual.Disposable#6](../../../samples/snippets/csharp/VS_Snippets_CLR/conceptual.disposable/cs/using5.cs#6)]
-[!code-vb[Conceptual.Disposable#6](../../../samples/snippets/visualbasic/VS_Snippets_CLR/conceptual.disposable/vb/using5.vb#6)]  
-  
+[!code-vb[Conceptual.Disposable#6](../../../samples/snippets/visualbasic/VS_Snippets_CLR/conceptual.disposable/vb/using5.vb#6)]
+
 You can follow this basic pattern if you choose to implement or must implement a `try/finally` block, because your programming language doesn't support a `using` statement but does allow direct calls to the <xref:System.IDisposable.Dispose%2A> method.
-  
+
+## IDisposable instance members
+
+If a class holds an <xref:System.IDisposable> implementation as an instance member, either a field or a property, the class should also implement <xref:System.IDisposable>. For more information, see [implementing the dispose pattern](implementing-dispose.md#implementing-the-dispose-pattern).
+
 ## See also
 
-- [Cleaning Up Unmanaged Resources](../../../docs/standard/garbage-collection/unmanaged.md)
+- [Cleaning up unmanaged resources](../../../docs/standard/garbage-collection/unmanaged.md)
```

Comment:
Do you plan to add a discussion of DisposeAsync anywhere?  Or is that already covered somewhere else?

Reply:
That's is a good question, I'm not seeing it explicitly covered elsewhere. It might be out of scope for this iteration, but I can add an issue to track that effort and prioritize it against the other items on the backlog. How important is it to discuss, and I'm assuming you'd be someone to get the details from?